R/correctionsataglance-data.R

```diff
@@ -203,7 +203,7 @@ parseCorrThresholds <- function(reportObject, timezone){
       type <- rep(threshold[['type']], times=nrow(periods))
       startDates <- periods[['startTime']]
       endDates <- periods[['endTime']]
-      value <- periods[['referenceValue']]
+      value <- if(isEmptyOrBlank(periods[['referenceValue']])){NA} else{periods[['referenceValue']]}
```

Comment:
@dpattermann-usgs the statement you're using to get T/F would need to return a vector the same length. But `isEmptyOrBlank` returns just one TRUE or FALSE so it wouldn't work in this particular scenario.